app/templates/partials/calendar/monthly_view/add_week.html

```diff
@@ -15,6 +15,15 @@
         {% for event in day.events %}
           <div class="{{ day.css['event'] }}"><b>{{ event[0] }}</b> {{ event[1] }}</div>
         {% endfor %}
+        {% for p in parasha %}
```

Comment:
`{% for parasha in parashot %}` is even better :)

Reply:
I updated :)